doc/07_performance-tips.md

```diff
@@ -0,0 +1,16 @@
+# Performance Tips
+
+* Disable VSync and set threading to Single Safe in the project settings. Terrain building and editing performs significnatly faster with these settings.
+
+* Excessive noise height will kill your performance, use with care. A height range of 300 can run at 300-400fps on a GTX1060, but a range of 6-10,000 might run at 30-40fps. Hopefully this will be addressed when occlusion culling is available in Godot 4.
+
+* Lower shadow distance. I might use 200-400 on a GTX1060, but maybe 20 or disabled on an integrated card.
```

Comment:
I wish I could tell the larger shadow maps to use only the lower LODs, but godot doesn't have any way to customize how shadows render so they always render the same amount of meshes the player would see from its point of view :shrug: